homeassistant/components/sensor/rmvtransport.py

```diff
@@ -90,6 +91,13 @@
                 next_departure.get(CONF_MAX_JOURNEYS),
                 next_departure.get(CONF_NAME),
                 timeout))
+
+    for sensor in sensors:
+        await sensor.async_update()
+
+        if not sensor.data.departures:
+            raise PlatformNotReady
+
     async_add_entities(sensors, True)
```

Comment:
Since we do the update manually above now, we can drop the second argument `True` here.

Reply:
I just have to try that new GitHub feature. ;-)
```suggestion
    async_add_entities(sensors)
```